app/setup/scss/_global.scss

```diff
@@ -0,0 +1,18 @@
+// global link definitions
+a,
+.link {
+  border-bottom: 1px solid $color-grayscale--600;
+  color: $color-grayscale--400;
+  font-family: $font-family--primary;
+  padding: 0 $spacing--5;
```

Comment:
- [ ] Should be `10px` according to design.

Reply:
Talked to Christoph about the spacing because I thought that 10px might be a little to much. He said, that 5px would be ok. Changed it back to 10px. What do you think?

Reply:
@silvanbaumann Please check with Christoph. If he says `5px` is ok, it is. He needs to decide on layout changes. If the value is changed, please tell him to also update the Element-Overview.